rust/src/blocks/physics/mod.rs

```diff
@@ -30,7 +30,7 @@ lazy_static! {
   static ref SIMULATION_TYPE: Type = {
     let mut t = common_type::object;
     t.details.object = CBTypeInfo_Details_Object {
-      vendorId: 0x73696e6b, // 'sink'
+      vendorId: 0x66726167, // 'frag'
```

Comment:
Shouldn't there be a global constant for that value?

Reply:
True